overlord/snapstate/handlers.go

```diff
@@ -196,7 +198,7 @@ func defaultPrereqSnapsChannel() string {
 	return channel
 }
 
-func linkSnapInFlight(st *state.State, snapName string) (bool, error) {
+func inFlightLinkSnap(st *state.State, snapName string) (*state.Task, error) {
```

Comment:
Maybe rename to `[find]LinkTaskForSnap()`?

I'm not sure what "in flight" means in this context, and whether it's helpful to have it in the function name.

Reply:
I think it's meant to imply that it only looks for a task that can still run, i.e., a non-Ready() task. But I agree, it's not very clear